packages/push-platform-notifications-controller/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@metamask/push-platform-notifications-controller",
+  "version": "0.0.0",
+  "description": "Manages push notifications for MetaMask decentralized authentication systems.",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/push-platform-notifications-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "tsup --config ../../tsup.config.ts --tsconfig ./tsconfig.build.json --clean",
+    "build:docs": "typedoc",
+    "changelog:update": "../../scripts/update-changelog.sh @metamask/push-platform-notifications-controller",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/push-platform-notifications-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest --reporters=jest-silent-reporter",
+    "test:clean": "jest --clearCache",
+    "test:verbose": "jest --verbose",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@firebase/app": "^0.10.4",
+    "@firebase/messaging": "^0.12.9",
+    "@metamask/base-controller": "^5.0.2",
+    "@metamask/notifications-controller": "^1.0.0",
+    "@metamask/profile-sync-controller": "^1.0.0",
+    "bignumber.js": "^4.1.0",
+    "deepmerge": "^4.2.2",
+    "loglevel": "^1.8.1"
+  },
+  "devDependencies": {
+    "@lavamoat/allow-scripts": "^3.0.2",
+    "@metamask/auto-changelog": "^3.4.4",
+    "@types/jest": "^27.4.1",
+    "@types/readable-stream": "^2.3.0",
+    "jest": "^27.5.1",
+    "ts-jest": "^27.1.4",
+    "typedoc": "^0.24.8",
+    "typedoc-plugin-missing-exports": "^2.0.0",
+    "typescript": "~4.9.5"
+  },
```

Comment:
Since this controller talks to `notifications-controller` and `profile-sync-controller`, it would be good to add them as peer dependencies:

``` suggestion
  },
  "peerDependencies": {
    "@metamask/notifications-controller": "^1.0.0",
    "@metamask/profile-sync-controller": "^1.0.0",
  },